docs/_guide/community.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: guide
+title: Community
+slug: community
+---
+
+{::options toc_levels="1..3" /}
+* ToC
+{:toc}
+
+
+There are many great resources and locations to learn about LitElement and web components,
+share what you've built, and more. All community participation is subject to the Polymer Project's
+<a href="https://github.com/Polymer/project/blob/master/Code_of_Conduct.md">Code of Conduct</a>—be
+excellent to each other!
+
+*   <strong>Join us on <a href="https://polymer-slack.herokuapp.com/">Slack</a>!</strong> 
+    Polymer Slack has thousands of users discussing LitElement, web components, and related topics.
+    Check out <code>#general</code>, <code>#beginners</code>, or
+    <code>#tools</code> to get started.
+
+*   Follow us on <a href="https://twitter.com/polymer">Twitter</a>
+    for the latest on web components and the Polymer Project. Many
+    of our team members can be found tweeting about LitElement, lit-html, 
+    and the latest developments in the web platform.
+
+*   The <a href="https://groups.google.com/forum/?fromgroups=#!forum/polymer-dev">polymer-dev</a>
+    mailing list is great for long-form questions and discussion on any of the libraries, polyfills, or 
+    standards the the Polymer Project works on.
+
+*   On <a href="https://stackoverflow.com/tags/polymer">StackOverflow</a> use
+    the <code>lit-element</code>, <code>lit-html</code>, or <code>shadow-dom</code>
```

Comment:
I'd go with web-component or nothing. Also these should be links.

Reply:
Sure. Yeah, it feels weird. So, when I searched before, I couldn't find any tags for web-component or custom-element, and I figured that was because people mostly had questions on shadow DOM.

Turns out I was just wrong. So I'd be inclined to say: Use the `lit-element` or `lit-html` tags. (You can also find questions and answers on the underlying web platform technologies using the `web-component`, `custom-element` and `shadow-dom` tags.)

This latter bit would only be on the LitElement page, of course.

Reply:
I think including `web-component` is good, just to reinforce that some questions might not be lit-element specific.

Reply:
Fixed.